src/Microsoft.Identity.Abstractions/DownstreamApi/IDownstreamApi.HttpMethods.tt

```diff
@@ -52,7 +58,10 @@ namespace Microsoft.Identity.Abstractions
      string user = !hasApp ? "user" : "null";
      string sampleCodeTemplate = template.Replace("TOutput", "IEnumerable<MyItem>").Replace("TInput", "MyItem").Replace("<", "&lt;").Replace(">", "&gt;");
 #>
+<# if (framework == "net8"){ #>
```

Comment:
You could actually move this line right after line 32, and avoid having a `#if` around each method in the generated code. 
Line 63 and 64 would have to go after line 147

Reply:
I think I've done what you are asking

Reply:
Thanks @dualtagh